CRM/Upgrade/Incremental/php/FiveSeventyFive.php

```diff
@@ -71,6 +71,40 @@ public function upgrade_5_75_alpha1($rev): void {
     $this->addTask('Replace receive_date smarty token in online membership template',
       'updateMessageToken', 'membership_online_receipt', '$receive_date', 'contribution.receive_date', $rev
     );
+
+    $this->addTask('Install tellafriend extension', 'installTellafriend');
+  }
+
+  /**
+   * Install tellafriend extension.
+   *
+   * This feature is restructured as a core extension - which is primarily a code cleanup step.
+   *
+   * @param \CRM_Queue_TaskContext $ctx
+   * @return bool
+   * @throws \CRM_Core_Exception
+   */
+  public static function installTellafriend(CRM_Queue_TaskContext $ctx) {
+    // Based on the instructions for the FiveThirty financialacls upgrade step
+    // Install via direct SQL manipulation. Note that:
+    // (1) This extension has no activation logic as of 5.75 (the DB tables are still in core)
```

Comment:
If the DB tables are still in core, then the xml/schema files should stay there too.